projects/addon-mobile/directives/sidebar/sidebar.directive.ts

```diff
@@ -42,14 +45,14 @@ export class TuiSidebarDirective<T = Record<string, unknown>>
         }
     }
 
-    constructor(
-        @Inject(TemplateRef) readonly content: TemplateRef<T>,
-        @Inject(Injector) private readonly injector: Injector,
-        @Inject(TuiDropdownService)
-        private readonly portalService: TuiDropdownService,
-        @Inject(ChangeDetectorRef) cdr: ChangeDetectorRef,
-    ) {
-        super(content, cdr);
+    readonly content: TemplateRef<T>;
+
+    constructor() {
```

Comment:
Shouldn't it just work without constructor at all?

Reply:
fixed